src/modules/GameConstants.ts

```diff
@@ -1357,6 +1358,12 @@ export const TemporaryBattles = [
     'Spectrier',
     'Dyna Tree Birds',
     'Regigigas',
+    'Aipom Alley',
+    'Mime Interview',
+    'Underground Fighting Ring',
```

Comment:
These needs to be in the same order as in the temporary battle list. And yes, that will require an update function, but you can just see how it's done above.
But yeah, it will be easier to read, if they are in the correct order

Reply:
Moved and added associated lines to update.ts